lnrpc/routerrpc/router_backend.go

```diff
@@ -205,24 +231,12 @@ func (r *RouterBackend) QueryRoutes(ctx context.Context,
 	// Query the channel router for a possible path to the destination that
 	// can carry `in.Amt` satoshis _including_ the total fee required on
 	// the route.
-	var (
-		route   *route.Route
-		findErr error
+	route, err := r.FindRoute(
+		sourcePubKey, targetPubKey, amtMSat, restrictions,
+		destTlvRecords, finalCLTVDelta,
```

Comment:
Can the `FindRoute` without cltv delta parameter now be removed?

Reply:
That's already done in the diff.

Reply:
Did you mean to ask why? If so, because the router's `FindRoute` already applies the default when the value is 0.

Reply:
I mean the `finalExpiry ...uint16` (three dots)